modules/git/diff.go

```diff
@@ -153,6 +153,7 @@ func CutDiffAroundLine(originalDiff io.Reader, line int64, old bool, numbersOfLi
 
 	for scanner.Scan() {
 		lof := scanner.Text()
+		fmt.Printf("%03d: %s\n", currentLine, lof)
```

Comment:
?

Reply:
Sorry forgot to pull before cherry picking

Reply:
```suggestion
```